flake.nix

```diff
@@ -69,10 +71,12 @@
             buildInputs =
               [
                 cairo
+                curl
                 freetype
                 gettext
                 imlib2
                 librsvg
+                libxml2
```

Comment:
Seems like appropriate include for [libxml2](https://search.nixos.org/packages?channel=23.11&show=libxml2&from=0&size=50&sort=relevance&type=packages&query=libxml2).